tpp-app/src/settings/SettingsNavigator.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import Settings from "./Settings"
+import Notifications from './Notifications';
+const Stack = createNativeStackNavigator();
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+export default function SettingsNavigator() {
+    return(
+        <Stack.Navigator intialRouteName="Settings" screenOptions={{ headerShown: true,
```

Comment:
When I ran there was a warning about multiple screen with duplicate names, should these be changed? Not too worried since its just a warning, but this could learn to confusion for the navigator 

<img width="200" alt="Screen Shot 2022-04-15 at 9 18 09 PM" src="https://user-images.githubusercontent.com/35851484/163656246-ef6265a0-1775-47fc-943e-e429ba1c8e62.png">